ui/css/itcss/components/newui-sections.scss

```diff
@@ -50,14 +50,6 @@
 // main-container media queries
 
 @include design-system.screen-sm-min {
-  .lap-visible {
-    display: flex;
-  }
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 54 26 AM](https://github.com/user-attachments/assets/1826da00-c6f3-4c81-8128-14201d8b1bf3)